docs/features/role-based-access-control/high-level-design.md

```diff
@@ -0,0 +1,100 @@
+# Role base access control - High-level design
+
+This document aims at providing an overview of the architecture of Role-based Access Control (RBAC from here on) for
+the Marxan platform.
+
+## Roles
+
+Platform administrators should be identified by a boolean attribute in the users table (`is_admin`). This attribute can
+only be edited by other platform administrators, and it must not be editable by the user when editing their profile.
```

Comment:
We shouldn't pack `user` table with something not related/needed for every resource. It quickly becomes overloaded with columns and hard to maintain.

More flexible approach would be "just" create a new dedicated table with only `userId` or even keep the as a common `role` type we already have.

Also not sure if this is necessary at this moment - we don't have particular requirements what platform admin can do (at least not said explicit) that would apply for the demo.

Reply:
Well, my idea was that a platform admin might actually impact every resource in the platform, hence why I added it to the `user` table. However, I don't see any problem with keeping a dedicated database table to store platform admins.

Regarding "not sure if this is necessary at this moment" see https://github.com/Vizzuality/marxan-cloud/pull/621#discussion_r744541550

Reply:
Changed implementation of platform admins to using a dedicated DB table for the purpose.

Also, added "development plan" document, identifying platform admins will only be implemented after the demo.